transcript/src/syncronized.rs

```diff
@@ -1,122 +0,0 @@
-use std::array;
-
-use crossbeam_channel::{Receiver, Sender, bounded};
```

Comment:
please also help to remove `crossbeam-channel` from `Cargo.toml`

Reply:
Done